vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -7,6 +7,7 @@ required_env_vars=(
     "LOCATION"
     "OS_TYPE"
     "SIG_IMAGE_NAME"
+    "SIG_GALLERY_NAME"
```

Comment:
Will it block Linux?

Reply:
Maybe?

Reply:
cc @ganeshkumar5699 for the change in the shared code.

Reply:
[https://github.com/Azure/AgentBaker/pull/2153/commits/c4f63ffd1705520c63e38a96b4546306bc8b93de](url)

**Context**: Changed gallery name in `sig_resource_id` from `PackerSigGalleryEastUS` to `SIG_GALLERY_NAME` because windows might need a different gallery destination for Gen 2 VHD.

**Sanity check**: Linux does not pass `SIG_GALLERY_NAME` to this file. So if `SIG_GALLERY_NAME` is empty, this file will set its value to `PackerSigGalleryEastUS` as in `cleanup.sh`.

Reply:
Could we close this?

Reply:
Yes since you have fixed it